packages/jsenv-plugin-commonjs/src/rollup_plugin_commonjs_named_exports.js

```diff
@@ -2,63 +2,71 @@
 
 import { readFileSync } from "node:fs"
 import { fileURLToPath, pathToFileURL } from "node:url"
+import path from "node:path"
 import { VM as VM2 } from "vm2"
 import resolve from "resolve"
 import isValidIdentifier from "is-valid-identifier"
 import { init, parse } from "cjs-module-lexer"
 
+const importWrapper = {
+  wrap: (specifier) => {
+    return `cjs_scan:${specifier}`
+  },
+  unwrap: (specifier) => {
+    return specifier.slice("cjs_scan:".length)
+  },
+  isWrapped: (specifier) => {
+    return specifier.startsWith("cjs_scan:")
+  },
+}
+
 export const rollupPluginCommonJsNamedExports = ({ logger }) => {
-  const inputSummaries = {}
-  const cjsScannedNamedExports = {}
+  const scanResults = {}
 
   return {
+    name: "scan_cjs_named_exports",
     async buildStart({ input }) {
       await init()
-
       Object.keys(input).forEach((key) => {
         const inputFilePath = input[key]
-        const inputFileUrl = pathToFileURL(inputFilePath)
```

Comment:
This is the fix for this lugin on windows (use the file path instead of an url). The rest is just adapting to this change, updating tests and other stuff